src/Components/Publishing/__tests__/Article.test.tsx

```diff
@@ -31,6 +31,17 @@ global.fetch = jest.fn(() =>
 jest.mock("../ToolTip/TooltipsDataLoader", () => ({
   TooltipsData: props => props.children,
 }))
+jest.mock("Artsy/Analytics/useTracking")
+
+const trackEvent = jest.fn()
+
+beforeEach(() => {
+  ;(useTracking as jest.Mock).mockImplementation(() => {
```

Comment:
For as often as we're repeating this block, it might make sense to extract a common function, so these all get reduced to something like 

```
beforeEach(() => {
  mockTracking(trackEvent)
})
```

It would be less repetition and less code to look at in each `beforeEach`.

Reply:
I actually tried creating an imported module with the repetitive code but it kept failing...when I ran in in the debugger the error thrown was  something like `the no- import-unmocked-threw an error` [which looks like an issue others have run into when importing mocks.](https://github.com/facebook/jest/issues/4262) I'd love to pair on this if you have some tijme, but I time-boxed and wasn't able to find a working solution during that allotted period.   Maybe its a good candidate for a Wednesday pairing session?